Content.Client/Stylesheets/IStylesheetManager.cs

```diff
@@ -1,12 +1,14 @@
+using Content.Client.Stylesheets.Redux;
 using Robust.Client.UserInterface;
 
-namespace Content.Client.Stylesheets
+namespace Content.Client.Stylesheets;
+
+public interface IStylesheetManager
 {
-    public interface IStylesheetManager
-    {
-        Stylesheet SheetNano { get; }
-        Stylesheet SheetSpace { get; }
+    Stylesheet SheetNanotransen { get; }
+    Stylesheet SheetSystem { get; }
+
+    public Dictionary<string, Stylesheet> Stylesheets { get; }
```

Comment:
i do it as a dictionary because `FancyWindow` has a `Stylesheet` property and needs to access a sheet by name, i could put that into a helper method maybe?

Reply:
Yes, do that.